pkg/mediationcontainer/remote_mediation_client.go

```diff
@@ -202,7 +205,7 @@ func getRequestType(serverRequest *proto.MediationServerMessage) RequestType {
 }
 
 type RequestHandler interface {
-	HandleMessage(serverRequest *proto.MediationServerMessage, probeMsgChan chan *proto.MediationClientMessage)
+	HandleMessage(serverRequest proto.MediationServerMessage, probeMsgChan chan *proto.MediationClientMessage)
```

Comment:
The errors encountered by the client implementations for discovery, validation  or action execution are converted to ErrorDTO or Action failed response by the TurboProbe. The message handlers will forward this message to the TurboServer.